apps/api-extractor/src/analyzer/Span.ts

```diff
@@ -61,8 +81,13 @@ export class SpanModification {
 
   /**
    * If true, then getModifiedText() will search for a "/*" doc comment in this span and indent it.
```

Comment:
This doc comment doesn't really match anymore

Reply:
Fixed